src/utils/roll.ts

```diff
@@ -88,10 +88,14 @@ export async function roll({
     const ref = `refs/heads/${branchName}`;
 
     d(`Checking that no orphan ref exists from a previous roll`);
-    const maybeOldRef = await github.git.getRef({ ...REPOS.electron, ref });
-    if (maybeOldRef.status !== 404) {
-      d(`Found orphan ref ${ref} with no open PR - deleting`);
-      await github.git.deleteRef({ ...REPOS.electron, ref });
+    try {
+      const maybeOldRef = await github.git.getRef({ ...REPOS.electron, ref });
+      if (maybeOldRef.status ===  200) {
+        d(`Found orphan ref ${ref} with no open PR - deleting`);
+        await github.git.deleteRef({ ...REPOS.electron, ref });
+      }
+    } catch (HttpError) {
```

Comment:
```suggestion
    } catch (e) {
      if (e instanceof HttpError)
        // ...
```
maybe...?

Reply:
happy to do whatever - i just took the same tack as the linked lines in the PR body

Reply:
weird, ok. i guess it's fine to just catch all errors and shove them in a variable called `HttpError`. Seems weird to me though 🤷‍♂